apps/web/src/features/organizations/components/Members/InvitesList.tsx

```diff
@@ -27,6 +29,13 @@ const headCells = [
 ]
 
 const InvitesList = ({ invitedMembers }: { invitedMembers: UserOrganization[] }) => {
+  const [deleteInvite] = useUserOrganizationsRemoveUserV1Mutation()
+  const currentOrgId = useCurrentOrgId()
+
+  const handleDeleteInvite = (invitedId: number) => {
+    deleteInvite({ orgId: Number(currentOrgId), userId: invitedId })
```

Comment:
I suggest wrapping this into a try catch. We can add error handling separately but a TODO would help.